wintaser/InputCapture.cpp

```diff
@@ -626,7 +626,7 @@ void InputCapture::ProcessInputs(CurrentInput* currentI, HWND hWnd){
 			continue;
 
 		// Now we build the SingleInput, and check if it's mapped to something.
-		SingleInput siPressed = {SINGLE_INPUT_DI_KEYBOARD, k, ""};
+		SingleInput siPressed = { SINGLE_INPUT_DI_KEYBOARD, static_cast<SHORT>(k), "" };

```

Comment:
It's perfectly ok to index an array with a `SHORT`, as long as the array isn't longer than the max value of a `SHORT`.